### PR TITLE
Fix #462

### LINKED ISSUE #462: Python binding not work with virtualenv
Before installing Triton, I switch to python virtual environment.
After that, I try to `import triton` from my virtual environment. 
It succeed once. ( The same session I install triton )
After closed the terminal, switch to virtual environment, and try to `import triton` again.
It failed.
I am not sure it's triton's problem, or the way I install triton is wrong.
Thanks.

### PR DESCRIPTION
**Pull request by bananaappletw**
Fix #462

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -326,9 +326,9 @@ if(PYTHON_BINDINGS)
         ${CMAKE_SOURCE_DIR}/src/testers/libTriton_units_testing.py
         IMMEDIATE @ONLY
     )
-    find_program(ENV_BINARY "env")
+    find_package(PythonInterp REQUIRED)
     if(UNIX)
-      install (CODE "execute_process(COMMAND ${ENV_BINARY} python2 ${CMAKE_BINARY_DIR}/setup.py install)")
+            install (CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_BINARY_DIR}/setup.py install)")
```
Comment (JonathanSalwan) [inline CMakeLists.txt]: What if the host have python2 and python3 installed and python3 is the default one?

File: src/scripts/setup.py.in [hunk 2]
```diff
@@ -53,7 +53,7 @@ setup(
     url                  = 'http://triton.quarkslab.com',
     zip_safe             = False,
     include_package_data = True,
-    data_files           = [([path for path in site.getsitepackages() if 'System' not in path][0], TRITON_MODULES)], # NOTE: Must be cleaner than this hack.
+    data_files           = TRITON_MODULES,
```
Comment (JonathanSalwan) [inline src/scripts/setup.py.in]: We got [warning](https://travis-ci.org/JonathanSalwan/Triton/jobs/191905345#L1274)

Comment (bananaappletw) [inline src/scripts/setup.py.in]: @JonathanSalwan Thanks for review. I will try to fix this problem soon. 😄

File: CMakeLists.txt (hunk 1 above)
Comment (bananaappletw) [inline CMakeLists.txt]: replace line to `find_package( PythonInterp 2.7 REQUIRED )`

File: src/scripts/setup.py.in (hunk 2 above)
Comment (bananaappletw) [inline src/scripts/setup.py.in]: This warning seems to happen in other build.
Maybe not related to this?
https://travis-ci.org/JonathanSalwan/Triton/jobs/191335294#L1251

**Pull request closed without merging**